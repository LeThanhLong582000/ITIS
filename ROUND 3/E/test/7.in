51
7925 8786
4601 4377
9323 3949
2304 966
4337 4689
361 1794
8021 5162
4001 1948
2885 5597
3501 6651
1145 9861
6241 6301
6361 6417
3392 8801
5651 6487
8971 9360
337 2821
3101 8216
2641 2833
7131 2337
3995 6876
137 7261
4616 9789
2975 5471
8953 7441
7731 7970
5311 6937
9131 5061
8545 1601
421 2033
2905 6010
3889 8065
6521 6480
6753 7483
8957 7451
8801 7265
6045 3785
6481 6801
2184 5529
6973 4781
7951 2444
7121 233
3501 8427
4301 5971
2371 7463
9084 41
8001 6257
1081 3139
661 1761
9901 1825
8001 585
