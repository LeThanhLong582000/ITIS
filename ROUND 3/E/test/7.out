1944043806
1225586064
-989726998
1462956856
-1838116032
-1482799914
821199654
1832885216
2025686416
1376201210
-139951396
-983703684
1637502280
-556131268
1979344196
1215615108
1835107956
-1003315660
-1050313760
1504397638
1266581312
1690146916
445516356
1771048656
-210238488
-1108377830
-1662239710
-237437966
-306396640
-1894253772
145913598
81709680
-901919476
1882600806
1064107914
-11727424
-202441500
-677692320
-508771788
-1133825128
-1592554256
-167469992
69009450
861128082
1716508164
1272337152
-255788176
-631168202
2019743060
1224597772
-794256920
